install/install_openadapt.ps1

```diff
@@ -181,15 +192,15 @@ function GetPythonCMD {
         $res = Invoke-Expression "python -V"
         $versionString = $res.Split(' ')[-1]
```

Comment:
Looks like there is some issue related to this line. (Noticed form 1st Screenshot)

Reply:
Hmm, I didn't touch the lines you referenced here, at all.

Reply:
Yes you are right, but we will have to fix it :)
Let us fix that in this pr only. If that is fine. :-)